packages/p2p-v2/src/components/Modals/NicknameModal/NicknameModal.scss

```diff
@@ -0,0 +1,48 @@
+.p2p-v2-nickname-modal {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
+    width: 440px;
+    padding: 2.4rem;
+    border-radius: 8px;
+    background: #fff;
+    box-shadow: 0px 32px 64px 0px rgba(14, 14, 14, 0.14);
+
+    @include mobile {
+        padding: 1.6rem;
+        width: 328px;
+    }
+
+    &__form {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        height: 100%;
+        font-size: 14px;
+
+        &-text {
+            padding: 1rem 0;
+        }
+
+        &-title {
+            padding-top: 1.6rem;
+        }
+
+        &__button-group {
+            display: flex;
+            justify-content: end;
+            margin-top: 1.4rem;
+            width: 100%;
+
+            & span {
+                font-size: 14px;
+            }
```

Comment:
can't this be handled by `Text` component?

Reply:
ah yes, will use text